go.mod

```diff
@@ -0,0 +1,12 @@
+module github.com/stelligent/crossing-go
+
+go 1.12
+
+require (
+	github.com/aws/aws-sdk-go v1.19.41
+	github.com/clunaslunas/crossing-go v0.0.5 // indirect
```

Comment:
I assume this is for testing - but we can't merge the PR with the wrong git repo named